dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/expand/ExternalFunctionExtensionCenter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.expand;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class ExternalFunctionExtensionCenter {
+
+    @Autowired
+    private TimePlaceholderResolverExpandService expandService;
+
+    private static ExternalFunctionExtensionCenter expandCenter;
+
+    @PostConstruct
+    public void init() {
+        expandCenter = this;
+        expandCenter.expandService = this.expandService;
+    }
+
+    public static boolean timeFunctionNeedExpand(String placeholderName) {
+        return expandCenter.expandService.timeFunctionNeedExpand(placeholderName);
+    }
+
+    public static String timeFunctionExtension(Integer processInstanceId, String timeZone, String placeholderName) {
+        return expandCenter.expandService.timeFunctionExtension(processInstanceId, timeZone, placeholderName);
+    }
+}
```

Comment:
Please remove `static`, this is not safe.

Reply:
whether a static method causes thread safety depends on whether a static field is used in the static method. If the static method does not operate a static field, only the instance field is used inside the method, which will not cause security problems.

Reply:
When someone execute `ExternalFunctionExtensionCenter.timeFunctionNeedExpand`, he will get unknow result, since he cannot sure if this class has been initialized. This is not a good practice, we need to avoid operating a non-static variable in a static method.

Reply:
And if someone uses this method in not spring object, he will get error.

Reply:
yes, this is indeed a matter of principle. At first, I thought that although the method call is inappropriate, it is thread-safe. I will make a suitable extension optimization for this, thank you for your review.